### ISSUE TITLE
Incorrect "days watched" value in AniDroid user profiles

### ISSUE THREAD

**Issue by Rainyan** [danejur/AniDroid#9]
Hi, first of all thanks for the great app. I've noticed a bug as follows:

AniDroid is displaying an incorrect "X days watched" value in users' profile pages.

Here is a screenshot (not my user profile, but this is replicable for all profiles with sufficiently large watch time values) from in-app, showing "61.4 days":

<img src="https://user-images.githubusercontent.com/6595066/220322482-2e59139b-13b2-4dc5-a005-379a37be8e33.png" width=300px alt="AniDroid screenshot" />

And here is the same data, accessed from a browser at https://anilist.co/user/1:

![daysWatched](https://user-images.githubusercontent.com/6595066/220323072-6356206e-77b7-4fcb-8345-b8d350bd0f8f.png)

Here's a [GraphQL query](https://anilist.co/graphiql?query=%7B%0A%20%20User%28id%3A%201%29%20%7B%0A%20%20%20%20siteUrl%0A%20%20%20%20%23%20Deprecated%0A%20%20%20%20stats%20%7B%0A%20%20%20%20%20%20watchedTime%0A%20%20%20%20%7D%0A%20%20%20%20%23%20Should%20use%20this%2C%20instead%0A%20%20%20%20statistics%20%7B%0A%20%20%20%20%20%20anime%20%7B%0A%20%20%20%20%20%20%20%20minutesWatched%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D) that reproduces this behaviour:

```graphql
{
  User(id: 1) {
    siteUrl
    # Deprecated
    stats {
      watchedTime
    }
    # Should use this, instead
    statistics {
      anime {
        minutesWatched
      }
    }
  }
}
```

which outputs:

```json
{
  "data": {
    "User": {
      "siteUrl": "https://anilist.co/user/1",
      "stats": {
        "watchedTime": 88410
      },
      "statistics": {
        "anime": {
          "minutesWatched": 91850
        }
      }
    }
  }
}
```
with `88410 / 60 / 24 ≈ 61.4 days (incorrect value from the deprecated API)`
and `91850 / 60 / 24 ≈ 63.8 days (correct value from the newer API)`

**Comment by danejur** [comment]
Good call-out! I'll update the User call to make use of those new fields. Thanks!
